CMakeLists.txt

```diff
@@ -83,7 +83,10 @@ specify custom installation with SCALAPACK_LIBRARIES or MKL_LIBRARIES variable")
 endif()
 
 if(ENABLE_ELSI)
-    if(NOT DEFINED ELSI_LIBRARIES)
+    if(DEFINED ELSI_LIBRARY)
+        add_library(elsi::elsi INTERFACE IMPORTED)
+        target_link_libraries(elsi::elsi INTERFACE "${ELSI_LIBRARIES}")
```

Comment:
```suggestion
        target_link_libraries(elsi::elsi INTERFACE "${ELSI_LIBRARY}")
```

Reply:
Yes, that way it makes indeed more sense, thanks a lot!